.circleci/config.yml

```diff
@@ -553,26 +557,73 @@ jobs:
+            a1981c0ab68b011eab1bb02f729e0dd7-3c9f20567ceb2efd.elb.us-west-2.amazonaws.com:80,\
+            a569b48655cbc40d092379f55e630966-e1f284c7eb8add73.elb.eu-west-1.amazonaws.com:80,\
+            aa4d66f81665240019022d50326c1654-cf61b8e99df12293.elb.eu-west-1.amazonaws.com:80,\
+            a5b46d6b8b23e42538ec0de5f0680350-46d80c995764cdd9.elb.us-west-2.amazonaws.com:80,\
+            a88662827a2d341bbb7beb883d6e32b9-2e166f46672c7f2b.elb.us-east-1.amazonaws.com:80,\
+            libra-fullnode-pre-mainnet.shopifyapps.com:80,\
+            20.50.43.42:80,\
+            34.68.134.0:80,\
+            35.193.177.28:80,\
+            3.230.173.177:32251' >> $BASH_ENV
+            #premainnet seed found here: https://github.com/libra/partners/blob/master/terraform/validator/package.sh#L13
+            echo 'export CONFIG_SEED_PREMAIN=4242613375bee239afd2e33a54e5133737a552fbf913bd37af133719e28a0012' >> $BASH_ENV
+            #premainnet number of nodes found here (login required)
+            #https://github.com/libra/partners/blob/master/helm/libra-validator/values.yaml#L2
+            echo 'export CONFIG_NUM_VAL_PREMAIN=15' >> $BASH_ENV
+            echo 'export CARGO_INCREMENTAL=0' >> $BASH_ENV
+            echo 'export CI_TIMEOUT="timeout 40m"' >> $BASH_ENV
+      - checkout
+      - run:
+          name: Get premain git revision
+          command: git checkout $GIT_REV_PREMIAN
+      - rust_setup
+      - print_versions
+      - env_setup
+      - install_deps
+      - run:
+          name: Build config-builder
+          command: |
+            cargo build -p config-builder
+      - run:
+          name: Build cluster-test
+          command: |
+            cargo build -p cluster-test
+      - run:
+          name: Generate premaintnet config key
+          command: |
+            cargo run -p config-builder -- faucet \
+            --output-dir "/tmp/config" \
+            --validators-in-genesis $CONFIG_NUM_VAL_PREMAIN \
+            --seed $CONFIG_SEED_PREMAIN
+      - run:
+          name: Premainnet cluster test
+          command: |
+            cargo run -p cluster-test -- --swarm --emit-tx --mint-file /tmp/config/mint.key --peers $NODES
+      - slack/status:
+          fail_only: true
+          webhook: "${WEBHOOK_PREMAINNET}"
+          failure_message: "<@channel> Cluster test failed on premainnet :ferris-detective:"
 workflows:
   commit-workflow:
     jobs:
-      - build-docker:
-        filters:
-          branches:
-            ignore: gh-pages
-      - validate-cluster-test-dockerfile:
-        filters:
-          branches:
-            ignore: gh-pages
-      - terraform:
-        filters:
-          branches:
-            ignore: gh-pages
-      - prefetch-crates:
-        filters:
-          branches:
-            ignore: gh-pages
+      - build-docker
+      - validate-cluster-test-dockerfile
+      - terraform
```

Comment:
Do we need these 3 jobs to run when building docs?

Reply:
the filter had no effect.   we need to put a small circle ci file with an ignore all in the gh-pages if we don't want circle ci to try to spin up (and immediately error out) on that branch.